index.html

```diff
@@ -204,7 +204,6 @@
           Promise&lt;void&gt; abort();
           Promise&lt;boolean&gt; canMakePayment();
 
-          readonly attribute DOMString? paymentRequestId;
```

Comment:
We should not treat `null` as a special case. It is no more special than `false` or `0` (all of them are "things that are vaguely negative, but are not the default-triggering special value of `undefined`").